Models/TrainingPrograms.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class TrainingPrograms
+    {
+        [Key]
+        public int TrainingProgramsId { get; set; }
+
+        [Required]
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+
+        [Required]
+        public string ProgramName { get; set; }
+        public string MaxAttendees { get; set; }
+
+        public List<TrainingPrograms> TrainingProgramsList = new List<TrainingPrograms>();
```

Comment:
This won't need a list of itself inside itself.

Reply:
But, it looks like it will need a list, of Employees who are enrolled in that program.
https://github.com/nashville-software-school/csharp-bangazonapi-boilerplate/issues/9